activerecord/test/cases/adapter_test.rb

```diff
@@ -442,6 +442,20 @@ def setup
       end
     end
 
+    def test_create_with_query_cache
+      @connection.enable_query_cache!
+
+      count = Event.count
+      binds = [[Event.column_for_attribute("id"), 1]]
+      bind_param = Arel::Nodes::BindParam.new(nil)
+      @connection.create("INSERT INTO events(id) VALUES (#{bind_param.to_sql})", nil, nil, nil, nil, binds)
+
+      assert_equal count + 1, Event.count
+    ensure
+      Event.find(1)&.destroy
```

Comment:
Can you use `reset_fixtures("posts")` and `Post` model instead?

```suggestion
      reset_fixtures("posts")
```

https://buildkite.com/rails/rails/builds/74129#3b2b1120-8262-4d6a-87b4-d6649ee9155d/1070-1079

Reply:
Or, use the returning value for `create` (`id = @connection.create(...)`).

```suggestion
      Event.find(id).destroy if id
```

Reply:
Done